### DISCUSSION TITLE
Bg_fill does not render correctly on WASM

### DISCUSSION THREAD

**Discussion by soundprojects** [emilk/egui#1333] [Q&A]
I have an Eframe template project where I want to change the background of a TextEdit

I use the following code using a utility method for converting the hex string to the proper RGB values:

```
        //Change button colors and text colors in fg_stroke
        style.visuals.widgets.inactive.bg_fill = Color32::from_hex("#4e505c").unwrap();
```

This makes the TextEdit render properly on desktop but sometimes on WASM, the background is still white

Is this a bug or am I missing something specific for WASM?

Thanks

**Comment by DusterTheFirst** [comment]
Could you provide screenshots of the problem?
